Description: Esc is show as key to cancel changed settings, but only "Q" works
Author: Axel Beckert <abe@debian.org>

Index: wicd/curses/wicd-curses.py
===================================================================
--- wicd.orig/curses/wicd-curses.py	2019-09-29 03:59:11.212526904 +0200
+++ wicd/curses/wicd-curses.py	2019-09-29 03:59:11.196526405 +0200
@@ -769,7 +769,7 @@
         ], self.handle_keys)
         self.confCols = OptCols([
             ('S', _('Save')),
-            ('esc', _('Cancel'))
+            ('Q', _('Cancel'))
         ], self.handle_keys)
 
     def lock_screen(self):
